### PR TITLE
Setup animation/#61

### PR DESCRIPTION
**Pull request by angely119**
- created Home component to render both Steps and Sprite components
- created Sprite component
- added sample sprite.png to assets folder
- installed Styled-components
- moved all styled-components to styles.js
- removed debugger

### CODE REVIEW

File: Healthkit.js [hunk 1]
```diff
@@ -1,7 +1,4 @@
-import AppleHealthKit, {
-  HealthValue,
```
Comment (angely119) [inline Healthkit.js:2]: removed unused components

File: Components/Sprite.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, { useState, useRef, useEffect } from "react";
+import { StyledSpriteContainer, StyledTileContainer, StyledSpriteImage } from "./styles";
+
+const Sprite = (props) => {
+  const { src, tile, scale, framesPerSprite, totalSprites } = props;
+  const [sprite, setSprite] = useState(0);
+  const requestIdRef = useRef();
+
+  useEffect(() => {
+    requestIdRef.current = requestAnimationFrame(animate);
```
Comment (angely119) [inline Components/Sprite.js:10]: requestAnimationFrame returns a request Id which is stored as a reference via the useRef hook

File: Components/Sprite.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    return () => cancelAnimationFrame(requestIdRef.current);
```
Comment (angely119) [inline Components/Sprite.js:11]: the requestId is passed into cancelAnimationFrame to cancels the animation frame request on unmount

File: Components/Sprite.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  }, [sprite]);
```
Comment (angely119) [inline Components/Sprite.js:12]: the useEffect hook runs every time sprite is changed

File: Components/Sprite.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      setSprite((sprite + 1) % totalSprites);
+    };
+    curFrame += 1;
+    requestIdRef.current = requestAnimationFrame(animate);
+  }
+
+  return (
+    <StyledSpriteContainer>
+      <StyledTileContainer width={tile.width} height={tile.height} scale={scale}>
+        <StyledSpriteImage source={src} left={tile.width * sprite} />
```
Comment (angely119) [inline Components/Sprite.js:27]: by setting the left margin of the image as the width of each sprite * the sprite number, it forces the tile to only show the current sprite

File: Components/Steps.js [hunk 6]
```diff
@@ -16,7 +16,6 @@ const Steps = () => {
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
```
Comment (angely119) [inline Components/Steps.js:19]: I had to remove this flex-grow setting to be able to render both the steps and sprite component in the same component

**Review by Conniec321 — APPROVED** [review]

**Review by kristinlam — APPROVED** [review]

**Pull request merged**